pinot-core/src/main/java/org/apache/pinot/core/startree/StarTreeUtils.java

```diff
@@ -48,14 +48,11 @@ public class StarTreeUtils {
   private StarTreeUtils() {
   }
 
-  public static final String USE_STAR_TREE_KEY = "useStarTree";
-
   /**
    * Returns whether star-tree is disabled for the query.
    */
   public static boolean isStarTreeDisabled(QueryContext queryContext) {
```

Comment:
(minor) This helper method does not add much value IMO. We may remove this method and directly call `queryContext.isSkipStarTree()`